app/styles/components/_logo.scss

```diff
@@ -1,7 +1,7 @@
-$logo-width: 179px;
-$logo-height: 120px;
+$logo-width: 149px;
+$logo-height: 100px;
 
-$logo-width-mobile: 76px;
+$logo-width-mobile: 75px;
```

Comment:
Confirmed with @fcbaila that the changes in the image dimensions should be done.